File: trail1/5의 배수의 합/sum-of-specific-condition-multiple.py
lst = list(map(int, input().split()))
A, B = lst[0], lst[1]
ssum = 0
if A > B:
    for i in range(B, A+1):
        if i % 5 == 0:
            ssum += i
else:
    for i in range(A, B+1):
        if i % 5 == 0:
            ssum += i
print(ssum)

File: trail1/5의 배수의 합/sum-of-specific-condition-multiple.cs
using System;

public class Codetree
{  
    public static void Main()
    {
        string[] arr = Console.ReadLine().Split();
        var(A, B) = (int.Parse(arr[0]), int.Parse(arr[1]));
        if (A > B) {
            int temp = B;
            B = A;
            A = temp;
        }
        int ssum = 0;
        for (int i=A; i<=B; i++) {
            if (i % 5 == 0) {ssum += i;}
        }
        Console.WriteLine(ssum);
    }
}
